docs/source/markdown/options/env-file.md

```diff
@@ -4,4 +4,9 @@
 ####> are applicable to all of those.
 #### **--env-file**=*file*
 
-Read in a line-delimited file of environment variables.
+Read the environment variables from the file, supporting prefix matching: `KEY*`, as well as multiline values in double quotes and single quotes, but not multiline values in backticks.
+The env-file will ignore comments and empty lines. And spaces or tabs before and after the KEY.
+If an invalid value is encountered, such as only an `=` sign, it will be skipped. If it is a prefix match (`KEY*`), all environment variables starting with KEY on the host machine will be loaded.
+If it is only KEY (`KEY`), the KEY environment variable on the host machine will be loaded.
+Compatible with the `export` syntax in **dotenv**, such as: `export KEY=bar`.
+To be compatible with Bash syntax, the `--env-file` will also support the `%%=` syntax, such as: `BASH_FUNC__echo%%=() { echo 1; }`.
```

Comment:
I am not sure about the last sentence? What is so special about that, I would expect any form of KEY=var to match so I don't see why this is worth a special notice.
Because it this form it is somewhat confusing what special bash syntax means.

Reply:
Okay. I removed it.